docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,374 @@
+    - ❗I am not working with other people.
+    - Share knowledge.
+    - Feedback about my writing.
+    - Feedback about my code.
+    - Formal code review.
+    - Integrate my code/ data with their downstream or upstream processes.
+    - Edit/ contribute some of their own code.
+    - Edit/ contribute some of their own writing.
+    - Teach/ tutor them.
+    - Peer programming.
+    - Deploy my code/ model/ pipeline/ dashboard.
+    - Other (please specify)
+
+18. What is the nature of your collaboration?
+    - Describe the collaboration:
+        - How long have you been working together?
+            - I am not collaborating.
+            - 2+ years.
+            - 1-2 years.
+            - 6-12 months.
+            - Less then 6 months.
+        - How frequently do you work together?
+            - I am not collaborating.
+            - 2+ times per week.
+            - Weekly.
+            - A few times a month.
+            - Monthly.
+            - Less then monthly.
+        - How do you divide the work?
+            - I am not collaborating.
+            - We work on different projects.
+            - We work on the same project, but different parts.
+            - We work on the same part of the same project together.
+    - Comments about collaboration:
+
+19. Do you have challenges with collaboration in Jupyter?
+    - Don't know what dependencies (versions of language, packages, extensions) a notebook uses.
+    - Don't know/ have the data a notebook is supposed to use.
+    - Poor support for our version control (git) system.
+    - No built-in way to publish my notebook to a shared location.
+    - Not being able to comment on notebooks.
+    - No "track changes;" can't figure out what changed between notebook checkpoints/ versions.
+
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+20. Do you have challenges with the notebook UI?
+    - No progress bar for running long notebooks.
+    - No marketplace for Extensions (e.g. 5 star ratings, browsable categories).
+    - No global search.
+    - Can't collapse sections of a notebook hierarchically.
+    - Poor autocompletion (e.g. LSP, show methods/ attributes).
+    - No modes for editing other Jupyter documents (MyST, Jupyter Book).
+    - Can't see hidden (.) files in file browser.
+    - Don't know which cell failed in long notebook.
+    
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+### You did it - thank you!
+
+21. Open feedback for problems/ pain points you didn't get to share.
+
+
+22. Optional - email address for recontact. Do we have your permission to recontact you for a user interview? 
+    We're looking for qualitative feedback on JupyterLab, JupyterHub, and the JupyterLab developer experience.
```

Comment:
Gathering emails requires trust, so I want to make sure it's clear enough to survey participants what we are asking them to do (user interview may not be a familiar term) and that leaving an email address is giving permission (right now the question asks for a yes/no but the answer space takes neither). I think something like this could work:

```suggestion
22. Optional - Are you interested in giving qualitative feedback on JupyterLab, JupyterHub, and the JupyterLab developer experience? If we have permission to contact you for follow-up questions, please leave your email address below.
```

Reply:
Good thinking. I'll swap out the text.